source/blog/2022-01-12-announcing-hanami-200alpha5.html.markdown

```diff
@@ -0,0 +1,206 @@
+---
+title: Announcing Hanami v2.0.0.alpha5
+date: 2022-01-12 11:00:00 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Sensible default logger, full source dirs config, router loading flexibility, route helpers in views
+---
+
+Happy new year, Hanami community! To get 2022 started, we're excited to announce the release of Hanami 2.0.0.alpha5!
+
+This release brings the last month of our work on Hanami 2.0 (with an extra week added for good measure, while we all returned from our end of year breaks). It includes:
+
+- Sensible default configuration for the application logger
+- Comprehensive `source_dirs` configuration
+- Lazy router and Rack app initialization
```

Comment:
```suggestion
- Lazy router and Rack app initialization (faster dev reloading)
```

Reply:
@jodosha n.b. this won't give us faster dev reloading _yet_ — there's still a missing piece of the puzzle, which is to have our app server started non-booted (and properly reloaded upon code changes) in dev-mode only — we don't have those yet.

What I might do is make a note that this opens up this possibility for us, and that we'll be making sure it's in place before 2.0.0 final.